app/timetables/admin.py

```diff
@@ -1,3 +1,5 @@
 from django.contrib import admin
-
+from app.timetables.models import Course
 # Register your models here.
```

Comment:
@andela-snwuguru  According to two scoops of Django,  It is advised we use explicit relative imports when importing from another module in the current app.
So instead of from `app.timetables.models import Course` let us use from .models import Course since models and admin sit under the same sub-app timetables